### PR TITLE
AgencyID MAC Name

### PR DESCRIPTION
**Pull request by russellbodine**

### CODE REVIEW

File: usaspending_api/accounts/models.py [hunk 1]
```diff
@@ -19,6 +19,8 @@ class FederalAccount(models.Model):
     agency_identifier = models.TextField(db_index=True)
     main_account_code = models.TextField(db_index=True)
     account_title = account_title = models.TextField()
+    # agency_identifier + '-' + main_account_code + ' - ' + account_title
```
Comment (nmonga91) [inline usaspending_api/accounts/models.py]: Some additional text here like "The federal_account_code field is made up of: ..." will make it clear it's describing the field below it

File: usaspending_api/references/reference_helpers.py [hunk 2]
```diff
@@ -23,6 +23,14 @@ def update_federal_accounts(tas_tuple=None):
         Number of rows updated
     """
 
+    # add federal_account_code
+    queryset = FederalAccount.objects.all()
```
Comment (nmonga91) [inline usaspending_api/references/reference_helpers.py]: How often will this be run?

Comment (russellbodine) [inline usaspending_api/references/reference_helpers.py]: any time loadtas is called, very rarely

File: usaspending_api/accounts/models.py (hunk 1 above)
Comment (russellbodine) [inline usaspending_api/accounts/models.py]: added text

File: usaspending_api/references/reference_helpers.py (hunk 2 above)
Comment (russellbodine) [inline usaspending_api/references/reference_helpers.py]: also it skips my code

**Pull request merged**